src/cpp/common/tls_credentials_options.cc

```diff
@@ -86,16 +84,43 @@ void TlsCredentialReloadArg::set_pem_root_certs(
 }
 
 void TlsCredentialReloadArg::add_pem_key_cert_pair(
-    TlsKeyMaterialsConfig::PemKeyCertPair pem_key_cert_pair) {
+    const TlsKeyMaterialsConfig::PemKeyCertPair& pem_key_cert_pair) {
   grpc_ssl_pem_key_cert_pair* ssl_pair =
       (grpc_ssl_pem_key_cert_pair*)gpr_malloc(
           sizeof(grpc_ssl_pem_key_cert_pair));
```

Comment:
Sorry for bringing this up late. How about refactor it (i.e., create a helper function in an anonymous namespace) so that it can be used in  `TlsCredentialReloadArg::set_key_materials` as well?

Reply:
Done.